common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -142,6 +147,8 @@ can_use_galactic_market = {
 	or = {
 		is_country_type = default
 		is_country_type = hfe_pirate
+		is_country_type = hfe_drone
```

Comment:
i dont see why drones should use galactic market

Reply:
This was done for flavor and a bonus to the nation that captures it.

I expanded some of the Harvesting Marvel's abilities and added in special Modules that represent its original purpose. Its purpose was to gather, process, sell, and provide resources for a Fallen Empire. There is a special Star Base in a special solar system that contains the Harvesting Marvel's economic computers.

Reply:
First, if you read contact event text, you will know this country has no ability to redistribute goods, it only collects it to warehouses.
Second, for my previous test, every country type gets it own galactic market eg pirates can affect prises only for other pirate countries.

Reply:
As I said before, I updated some of its story. Also, it no longer reports market transactions because it is selling them aimlessly and not transferring them to its master.

Reply:
As i said drone country type can interference default country market.
It make no sense.

Reply:
When the Galactic Market opens, everyone listed in the Country Types and this Game Rule can use it and interact with it. It's a part of the extension of the story and the new Modules that I added.

Reply:
Did you read what i write above?

Reply:
Why should Pirates use the Galactic Market, then? It would be better not to overwrite it at all.

Reply:
Okay, done

Reply:
Lets remove it then.

Reply:
Agreed.